src/providers/currency/coin.ts

```diff
@@ -63,7 +63,7 @@ export const availableCoins: CoinsMap<CoinOpts> = {
     paymentInfo: {
       paymentCode: 'BIP73',
       protocolPrefix: { livenet: 'ducatus', testnet: 'ducatus' },
-      ratesApi: 'https://bitpay.com/api/rates',
+      ratesApi: 'https://rates.ducatuscoins.com/api/v1/rates/',
```

Comment:
Надо добавить ApiProvider и указать откуда брать рейты
`const apiProvider: ApiProvider = new ApiProvider();`
> можно инициализировать  перед
`export const availableCoins: CoinsMap<CoinOpts> = {`

```suggestion
      ratesApi: apiProvider.getAddresses().ratesApi,
```